### ISSUE TITLE
Warning : very risky to use

### ISSUE THREAD

**Issue by houdayec** [tntkhang/gmail-sender-helper#1]
Hi everyone, Hi @tntkhang.

This post is written on 12th December, just to make people know about an important issue with this version.

To use this Gmail email sender, you need to use your email address and password (it makes sense).

`GMailSender.withAccount("your-email@gmail.com", "email-password")`

However, this is a huge problem, I explain myself :

If you put your password in a non-encoded format as it is specified in order to user this helper, be aware that if you release your APK on the public Google Play Store, it will be possible to analyse your APK.

I'm often analyzing APK in order to find some security breaches, and for sure I'll be able to see in a clear format both email and password, meaning that I'll have full access to your gmail account (and maybe more services if you use this password somewhere else).

So far, I really advice you either to not use this version of helper until @tntkhang find a way to fix it or to use a simple password on a brand new gmail account that doesn't really matter to you if it gets hack (but still very risky).

I don't post it to destroy this repo because basically if I'm there, it's because it's filling up one of my need, but I wouldn't use this version.

**Comment by AdelKIHAL** [comment]
Maybe you can store your user and password in firebase (for example) and retrieve them at runtime

**Comment by houdayec** [comment]
> Maybe you can store your user and password in firebase (for example) and retrieve them at runtime

Yes, it might be a good way, needs maybe to be secured with Firebase auth token with short expiration for instance.
